src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/MetricItemExtensions.cs

```diff
@@ -256,6 +257,11 @@ internal static OtlpMetrics.Metric ToOtlpMetric(this Metric metric)
                             AddAttributes(metricPoint.Tags, dataPoint.Attributes);
                             dataPoint.Count = (ulong)metricPoint.GetHistogramCount();
                             dataPoint.Sum = metricPoint.GetHistogramSum();
+                            if (metric.MetricType == MetricType.HistogramWithMinMax)
```

Comment:
Add a CHANGELOG entry for this.

Reply:
This needs the CHANGELOG of OTLP Exporter to be updated.

Reply:
Ahh, I didn't realise you meant OTLP Exporter CHANGELOG originally. Added.